source/unified-test-format/unified-test-format.rst

```diff
@@ -340,13 +340,9 @@ The top-level fields of a test file are as follows:
 
 .. _initialData:
 
-- ``initialData``: Optional array of one or more `collectionData`_ objects. Data
-  that will exist in collections before each test case is executed.
-
-  Before each test and for each `collectionData`_, the test runner MUST drop the
-  collection and insert the specified documents (if any) using a "majority"
-  write concern. If no documents are specified, the test runner MUST create the
-  collection with a "majority" write concern.
```

Comment:
I've noted that this is redundant given what is already specified in "Executing a Test".